Documentation/guides/building-apps/build-items.md

```diff
@@ -14,6 +14,23 @@ ms.date: 07/26/2022
 Build items control how a Xamarin.Android application
 or library project is built.
 
+## AndroidAdditionalJavaManifest
+
+`<AndroidAdditionalJavaManifest>` is used in conjunction with [Java Dependency Resolution](../JavaDependencyVerification.md).
+
+It is used to specify additional POM files that will be needed to verify dependencies.
+These are often parent or imported POM files referenced by a Java library's POM file.
+
+```xml
+<ItemGroup>
+  <AndroidAdditionalJavaManifest Include="mylib-parent.pom" JavaArtifact="com.example.mylib-parent" JavaVersion="1.0.0" />
+</ItemGroup>
+```
+See the [Java Dependency Resolution documentation](../JavaDependencyVerification.md)
+for more details.
+
+This build action was introduced in .NET 9.
```

Comment:
This section should explicitly call out supported item metadata, a'la https://github.com/xamarin/xamarin-android/blob/9f548d50cc03e24b85fe777a183251b184acdd93/Documentation/guides/building-apps/build-items.md#javasourcejar

> …the following MSBuild metadata is supported:
> 
> * `%(CopyrightFile)`: A path to a file that contains copyright
>     information for the Javadoc contents, which will be appended to
>     all imported documentation.
> 
> * `%(UrlPrefix)`: A URL prefix to support linking to online
>     documentation within imported documentation.
> 
> * `%(UrlStyle)`: The "style" of URLs to generate when linking to
>     online documentation.  Only one style is currently supported:
>     `developer.android.com/reference@2020-Nov`.

Is `%(Repository)` supported?

Reply:
This also should be done with `@(AndroidMavenLibrary)`, which should also mention/document `%(AndroidMavenLibrary.Repository)`.

Reply:
For consideration: should item metadata instead be documented in *tables*, a'la [MSBuild `<Exec/>` Task documentation](https://learn.microsoft.com/en-us/visualstudio/msbuild/exec-task?view=vs-2022)?